src/corg/working-on-nuxeo-sources/compiling-nuxeo-from-sources.md

```diff
@@ -112,7 +112,7 @@ history:
 ---
 ## Requirements
 
-*   [Sun Java](http://java.sun.com/) 6 (up to Nuxeo Platform 5.6), 7 (since Nuxeo Platform 5.7) or 8 (since Nuxeo Platform 7.2)
+*   [Sun Java](http://java.sun.com/) 11 (since Nuxeo Platform 11.1), 8 (since Nuxeo Platform 7.2), 7 (since Nuxeo Platform 5.7) or 6 (up to Nuxeo Platform 5.6)
```

Comment:
As it's being updated, couldn't we:
- Remove the references to JDK  and 6 and 7 and the old Platform versions.
- Remove the reference to Sun and align on https://doc.nuxeo.com/nxdoc/installation/#hardware-and-software-requirements: "Oracle's JDK or OpenJDK".

Reply:
We should just say OpenJDK really, that's our real requirement.